### ISSUE TITLE
Erratic Transaction behaviour

### ISSUE THREAD

**Issue by joshpitzalis** [CryptoMinerWorld/auction#32]
Transactions sometimes go though and don't other times. 

The fact that they can go through means that the front end is not broken. 

What other factors would cause the transaction to jump around in gas estimates and sometimes not go through?

**Comment by joshpitzalis** [comment]
This is what happens when try to cancel my glitched Halloween gem auctions. And before a different transaction, and error popped up. I will share a screen shot of that, if I can get it to happen again.
Pasted image at 2018-11-03, 10:51 PM 



John_Butler [9:03 AM]
This happened when I tried listing a gem for sale.
But I also just listed another gem right before this happened. The transaction was taking quite a few minutes even though I was using gwei way over the recommended price(etherscan recommended 3 I was using 6). After waiting like 5 minutes. I upped the price a bit (6.6) and the transaction was confirmed in less than a minute. But the gem page never updated. I never changed pages, but I guess the creating a new transaction with more gwei breaks the site too, and the reconciliation system will catch this.
Pasted image at 2018-11-03, 11:33 PM 


John_Butler [9:09 AM]
I refreshed the page and now the transaction is allowed... I remember this unstable action was here from the start. (edited)

John_Butler [9:26 AM]
I just kept refreshing this page and clicking the Create Auction button. It worked for 2 times and on the 3rd refresh this error popped up
Pasted image at 2018-11-03, 11:55 PM 


John_Butler [9:41 AM]
This is what happens when I try to cancel an auction
Pasted image at 2018-11-04, 12:11 AM 


John_Butler [9:47 AM]
Just so you know When I listed this new gem, the transaction went through. So that problem I mentioned was just because I upped the gwei price. That breaks the system. Which in it self is not good, but that should be fixed with the new transaction system you are building, I think.
Pasted image at 2018-11-04, 12:18 AM 


John_Butler [9:54 AM]
And results seem to change sometimes. (when trying to remove gem from auction) (edited)
Pasted image at 2018-11-04, 12:23 AM 

And here is yet another oddity, this happened when I tried to remove the gem from auction
Pasted image at 2018-11-04, 12:28 AM 


John_Butler [10:03 AM]
I even get the different errors without refreshing the page. Just clicking End Auction then reject the MetaMask transaction and click it again.
